features/kubernetes-nmstate/README.md

```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
 For testing you can use the `./testing.sh` script in this folder.
 The test will create a bond interface with the name `bond1` and attach `ens8` and `ens9` as the slave interfaces.
 
-*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
\ No newline at end of file
+*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
+
+### Node configuration examples
+
+```
+# IP configuration
+
+10: bond1: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 1c:40:24:99:5d:0d brd ff:ff:ff:ff:ff:ff
+    inet 192.168.10.100/24 brd 192.168.10.255 scope global noprefixroute bond1
+       valid_lft forever preferred_lft forever
+11: bond1.20@bond1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 1c:40:24:99:5d:0d brd ff:ff:ff:ff:ff:ff
+    inet 192.168.20.100/24 brd 192.168.20.255 scope global noprefixroute bond1.20
+       valid_lft forever preferred_lft forever
+
+# Bond configuration
+
+Ethernet Channel Bonding Driver: v3.7.1 (April 27, 2011)
+
+Bonding Mode: load balancing (round-robin)
+MII Status: up
+MII Polling Interval (ms): 140
+Up Delay (ms): 0
+Down Delay (ms):
+
+Slave Interface: enp3s0f2
+MII Status: up
+Speed: 1000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: 1c:40:24:99:5d:0d
+Slave queue ID: 0
+
+Slave Interface: enp3s0f3
+MII Status: up
+Speed: 1000 Mbps
+Duplex: full
+Link Failure Count: 0
+Permanent HW addr: 1c:40:24:99:5d:0e
+Slave queue ID: 0
+
```

Comment:
Unneeded new line

Reply:
This is to be expected, reflecting command output here.